src/types/publication.types.ts

```diff
@@ -47,6 +47,7 @@ export interface Publisher {
 }
 
 export const emptyPublisher: Publisher = {
+  type: '',
```

Comment:
burde ikke dette være `BackendTypeNames.PUBLISHER`?

Reply:
Denne skal bare settes av kanalregisterproxy: https://unit.atlassian.net/browse/NP-778

Reply:
tror egentlig emptyPublisher burde kuttes helt, siden det bare er null hvis ingenting er valgt(?), men det er en annen sak

Reply:
men uansett bedre at vi setter typen i tillegg her da. Hvertfall i mellomtiden før den tasken der er fikset. Nå er jo `type: PublicationContext` satt hardkodet i `PublisherField.tsx`

Reply:
fjernet det i denne PRen. Men vi bør kanskje ikke merge dette før det er på plass nei, så skulle egentlig vært en draft dette...

Reply:
Denne saken tilfører uansett ikke noe særlig nytt, den bare setter type for instance og context, sånn at vi kan håndtere det når backend ganske snart krever det

Reply:
Legger du på `BackendTypeNames.PUBLICATION_CONTEXT`?

Reply:
hm, denne kan mange ulike verdier da, som kommer fra backend. Foreløpig er det bare "Journal", men på sikt blir det også Book, og flere(?). Så vi må i så fall inn å endre her hver gang backend returnerer en ny type da

Begynte å fjerne hele emptyPublisher (siden verdien bør være null når ingenting er valgt), men det var litt mer jobb enn jeg trodde så lot det bare ligge

Reply:
åja, ok. men så lenge vi får lagret så er det vel greit 💯